dotcom-rendering/src/web/browser/ga/ga.ts

```diff
@@ -193,6 +193,22 @@ export const trackNonClickInteraction = (actionName: string): void => {
 	}
 };
 
+export const trackSponsorLogoLinkClick = (sponsorName: string): void => {
+	const { ga } = window;
+
+	if (ga) {
+		ga(send, 'event', 'click', 'sponsor logo', sponsorName, {
```

Comment:
To add some extra visibility, here we're recreating the logic that already exists in frontend (https://github.com/guardian/frontend/blob/main/static/src/javascripts/projects/common/modules/analytics/google.ts#L45). To us, It looked like this logic had never been migrated over to DCR which was why the expected events weren't being sent.